client/src/components/common/UserMenu/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { profilePath, settingsPath, logoutPath } from './pathData';
+import './index.css';
+
+const UserMenu = ({ username }) => (
+  <div className="user-menu">
+    <nav className="user-menu__nav">
+      <a href={`/profile/${username}`} className="user-menu__nav--link">
```

Comment:
I believe you want to use React router's Link component instead of standard HTML links https://reacttraining.com/react-router/web/example/basic

Reply:
Did it, can you review it again? @11janci